### PR TITLE
Project 5 -weather app- Jessica Panditha

### PR DESCRIPTION
**Pull request by jespa374**
Link to my deployed project:
https://weatherforecastbyj.netlify.app/

My reflections on how this week's project turned out:
This project was tough for me and it took a while for me to learn how to extract the data from the APIs, especially the image icons. After a while I figured it out. Also looping through arrays inside fetch functions and formatting date seem difficult for me but I guess it'll come with time and more practice. I have tested the site on the different breakpoints stated in CSS in Chrome devtools. 

Things I'd like to have clarified or explained in more detail:
Sometimes I experience that the current weather image looks different based on what image shows for example if a cloud shows instead of a sun so when I set a width and or/height the image looks good for that moment but when the image changes the sizes set from the beginning doesn't match the current image which makes it difficult to give set a good size. i don't know if you have any tips in solving this issue.

### CODE REVIEW

**Review by sofiavazs — APPROVED** [review]
Awesome job Jessica! 👏🏻
Your code is super clean, easy to read and nicely structured! It was a pleasure to review it! 
Even with time constraints that you mentioned on your READme I think you did a very good job my making it responsive, with different versions for mobile and tablet so it's more user friendly and retrieving more information details about the current weather (like humidity, feels like temperature and wind speed). 
I can tell you have a very good understanding how to fetch data from the API, manipulate it and changing the DOM either by manipulating the inner HTML or injecting it from JavaScript.
You should be really proud and keep up the good work! 💃

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
```
Comment (EmOlsz) [inline code/index.html:6]: Let's not forget about `./` in path to file

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <h3 id="sunset-time"></h3>
+        <h3 id="min-temp"></h3>
+        <h3 id="max-temp"></h3>
+    </div>
+    <section class="forecast-weather">
+        <p class="forecast-day" id="forecast-day"></p>
+        <p class="forecast-feels-like-temp" id="forecast-feels-like-temp"></p>
+        <div class="forecast-weather-icon"id="forecast-weather-icon"></div> 
+    </section>
+    <script src="script.js"></script>
```
Comment (EmOlsz) [inline code/index.html:33]: `./`

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+const windSpeed = document.getElementById('wind-speed');
+const feelsLikeTemp = document.getElementById('feels-like-temp');
+const minTemp = document.getElementById('min-temp');
+const maxTemp = document.getElementById('max-temp');
+const sunriseTime = document.getElementById('sunrise-time');
+const sunsetTime = document.getElementById('sunset-time');
+const currentDay = document.getElementById('current-day');
+const currentWeatherIcon = document.getElementById('current-weather-icon');
+const today = new Date();
+const now = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
```
Comment (EmOlsz) [inline code/script.js:20]:
```
const now = `${today.getFullYear()} - ${today.getMonth() + 1} - ${today.getDate()}`
```

Looks better , don't you think? ;)

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+//five-day forecast
+
+fetch(forecast_API_URL)
+.then(response => {
+    return response.json();
+})
+.then((forecastObject) => {
+    const listArray = forecastObject.list;
+    const filteredForecast = listArray.filter(item => 
+    item.dt_txt.includes('12:00'));
```
Comment (EmOlsz) [inline code/script.js:62]: Wrong indentation

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    forecastFeelsLikeTemp.innerHTML += `<p>${temp}° </p>`;
+    forecastWeatherIcon.innerHTML += `<img src="https://openweathermap.org/img/wn/${day.weather[0].icon}.png" />`;
+    }); 
+}); 
+       
+      
+        
+       
+
+
```
Comment (EmOlsz) [inline code/script.js:80]: Let's remove those empty lines

File: code/style.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
+.header {
+    display: flex;
+    justify-content: center;
+}
+
+.city {
+    font-size: 60px;
+    font-family: helvetica;
```
Comment (EmOlsz) [inline code/style.css:8]: You add `helvetica` as font-family in many places. To keep it clean, you could add it just one time, to html or body perhaps, like that :

```
body {
    font-family: helvetica;
}
```

File: code/style.css [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    .current-weather-wrapper h3 {
+        font-size: 25px;
+    }
+}
+
+/*Styling for desktop*/
+@media (min-width: 900px) {
+    .header {
+    background: rgb(6,6,20);
+    background: linear-gradient(90deg, rgba(6,6,20,0) 0%, rgba(252,252,36,0.9782797987616099) 98%, rgba(209,218,106,1) 100%, rgba(80,43,19,1) 100%, rgba(154,79,19,1) 100%, rgba(43,142,154,1) 100%, rgba(18,28,72,1) 100%, rgba(8,27,135,1) 100%, rgba(16,28,87,1) 100%, rgba(12,28,111,1) 100%, rgba(22,29,47,0.9266894257703081) 100%, rgba(35,180,207,1) 100%);
```
Comment (EmOlsz) [inline code/style.css:82]: Wrong indentation

**Review by EmOlsz — APPROVED** [review]
Good job Jessica! Your project is clean and well done, congratulations :) I can see that from time to time you have tendency to complicate your code more than you should, but it's not big problem - take a look on my comments, I hope they will help you improve!

**Pull request closed without merging**